uploadNeuroDB/tarchiveLoader

```diff
@@ -22,43 +22,42 @@ perl uploadNeuroDB/tarchiveLoader </path/to/DICOM-tarchive> C<[options]>
 
 Available options are:
 
--profile                    : Name of the config file in
-                              C<../dicom-archive/.loris_mri>
+-profile                 : Name of the config file in C<../dicom-archive/.loris_mri>
 
--force                      : Force the script to run even if the validation
-                              has failed
+-force                   : Force the script to run even if the validation
+                           has failed
 
--reckless                   : Upload data to database even if study protocol is
-                              not defined or violated
+-reckless                : Upload data to database even if study protocol is
+                           not defined or violated
 
--globLocation               : Loosen the validity check of the tarchive allowing
-                              for the possibility that the tarchive was moved to
-                              a different directory
+-globLocation            : Loosen the validity check of the tarchive allowing
+                           for the possibility that the tarchive was moved to
+                           a different directory
 
--noJIV                      : Prevent the JIVs from being created
+-noJIV                   : Prevent the JIVs from being created
 
--newScanner                 : By default a new scanner will be registered if the
-                              data you upload requires it. You can risk turning
-                              it off
+-newScanner              : By default a new scanner will be registered if the
+                           data you upload requires it. You can risk turning
+                           it off
 
--keeptmp                    : Keep temporary directory. Make sense if have
-                              infinite space on your server
+-keeptmp                 : Keep temporary directory. Make sense if have
+                           infinite space on your server
 
--xlog                       : Open an xterm with a tail on the current log file
+-xlog                    : Open an xterm with a tail on the current log file
 
--verbose                    : If set, be verbose
+-verbose                 : If set, be verbose
 
--seriesuid                  : Only insert this SeriesUID
+-seriesuid               : Only insert this SeriesUID
 
--acquisition_protocol       : Suggest the acquisition protocol to use
+-acquisition_protocol    : Suggest the acquisition protocol to use
 
--bypass_extra_file_checks   : Bypass extra_file_checks
+-bypass_extra_file_checks: Bypass extra_file_checks
```

Comment:
perhaps the function name can be in C<>? (extra_file_checks)

Reply:
done